snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -95,15 +95,15 @@ public void CreateBucket()
         {
             var projectId = _fixture.ProjectId;
 
-            // <Buckets.Insert>
+            // <Service.Buckets.Insert>
```

Comment:
I’m in favor of including all of the reference docs, but we’ll have to figure out a way to lay them out in layers.